src/components/MapContainer.js

```diff
@@ -80,12 +80,12 @@ class MapContainer extends React.Component {
       .then(coordinates => getTraceAttributes(host, coordinates))
       // This `catch` statement is placed here to handle errors from Fetch API.
       .catch(error => {
+        this.props.stopLoading()
```

Comment:
Right, I think we might need to think through what are errors we would recover from and try to keep working whenever possible, versus errors that would need to stop everything because we can't continue. In the case of routing failure, we would not actually get anything usable from the server, so we have to bail on everything. In the case of missing data tiles, we can try to keep going with data we have. One case we might end up having to bail on is if we try to get data tiles and _none_ of the data we try to get ever succeeds.